test/y2storage/devicegraph_test.rb

```diff
@@ -391,19 +391,21 @@ def with_sda2_deleted(initial_graph)
 
       it "does not perform a system lookup" do
         expect(Y2Storage::BlkDevice).to_not receive(:find_by_any_name)
+        devicegraph.find_by_any_name("/dev/sda1")
```

Comment:
:+1: Good catch about these broken tests!... which is kind of embarrassing, by the way